apps/testing/src/main/java/com/akto/testing/ApiExecutor.java

```diff
@@ -201,6 +205,44 @@ private static OriginalHttpResponse getRequest(OriginalHttpRequest request, Requ
         return common(okHttpRequest, followRedirects, debug, testLogs);
     }
 
+    private static RequestBody getFileRequestBody(String fileUrl) throws Exception{
+        try {
+            URL sourceFileUrl = new URL(fileUrl);
+            InputStream urlInputStream = sourceFileUrl.openStream();
+            final int CHUNK_SIZE = 1 * 1024 * 1024 ;
+
+            RequestBody requestBody = new MultipartBody.Builder()
+                .setType(MultipartBody.FORM)
+                .addFormDataPart("file", "filename", new RequestBody() {
+                    @Override
+                    public MediaType contentType() {
+                        return Utils.getMediaType(fileUrl);
+                    }
+
+                    @Override
+                    public void writeTo(BufferedSink sink) throws IOException {
+                        byte[] chunk = new byte[CHUNK_SIZE];
+                        int bytesRead;
+                        long totalBytesRead = 0; 
+                        long maxBytes = 100L * 1024 * 1024; 
+                        while ((bytesRead = urlInputStream.read(chunk)) != -1) {
+                            totalBytesRead += bytesRead;
+                            if (totalBytesRead > maxBytes) {
+                                throw new IOException("File size exceeds the maximum limit of 100MB");
```

Comment:
instead of throwing exception, let's exit from the function, so that we just upload 100Mb of data at max

Reply:
done